hydrus/resources.py

```diff
@@ -513,8 +512,8 @@ def put(self, path, int_list="") -> Response:
                     if not check_required_props(class_path, obj):
                         incomplete_objects.append(obj)
                         object_.remove(obj)
-                link_props_list, link_type_check = get_link_props_for_multiple_objects(class_path,
-                                                                                       object_)
+                link_props_list, link_type_check = get_link_props_for_multiple_objects(class_path, object_)
```

Comment:
@thrishik7 You are getting the `E501 line too long (111 > 100 characters)` from here. Try to undo this change to fix this error.

Reply:
okay @sameshl I will try.